plone/recipe/codeanalysis/__init__.py

```diff
@@ -77,6 +79,9 @@ def __init__(self, buildout, name, options):
         # CSS Lint
         self.options.setdefault('csslint', 'False')
         self.options.setdefault('csslint-bin', 'csslint')
+        # check-manifest
+        self.options.setdefault('check-manifest', 'True')
```

Comment:
check-manifest takes ~10 seconds for `plone.recipe.codeanalysis` package on my machine, it's the longest running check, so from my point of view we should be able to disable it. And i'm thinking about writing a cache file which detect if a file was added/removed so check-manifest could be rerun, otherwise it's just annoying to have it as pre-commit hook, because it takes too much time.